### PR TITLE
FSL-1.1

### PR DESCRIPTION
**Pull request by chadwhitacre**
Closes https://github.com/getsentry/fsl.software/issues/23.

Two biggest changes:

# Permitted Purpose

one

# Grant of Future License

two

### CODE REVIEW

File: FSL-1.1-Apache-2.0.template.md [hunk 1]
```diff
@@ -30,18 +30,15 @@ and redistribute the Software for any Permitted Purpose identified below.
...
 
-1. as a substitute for any of our products or services;
+2. offers the same or substantially similar functionality as the Software; or
 
-2. in a way that exposes the APIs of the Software; and
-
-3. in a product or service that offers the same or substantially similar
-   functionality to the Software.
+3. otherwise competes with the Software or any other product or service we
+offer using the Software as of the date we make the Software available.
```
Comment (chadwhitacre) [inline FSL-1.1-Apache-2.0.template.md:41]: > seems weird to use "competes" in a definition of what competing is, no?

**Comment by chadwhitacre** [comment]
Last call for comments! I've squashed changes and also applied to MIT. Here is the diff: https://github.com/getsentry/fsl.software/pull/32/commits/6a886da97c12b28d48f7f974c4d5b9695fe312cd.

@RhysSullivan @schacon @rohitpaulk Yinz early adopters want to give this diff a once over to make sure nothing surprises you? Ideally this is just clarifying the intent of the FSL as you already understood it. 🙏

**Comment by chadwhitacre** [comment]
[Last call on Twitter](https://twitter.com/chadwhitacre_/status/1757902884405227903).

**Comment by RhysSullivan** [comment]
Looks good 👍

**Comment by rohitpaulk** [comment]
Looks good to me!

**Comment by chadwhitacre** [comment]
Shipping in 5 ... 4 ... 3 ... 2 ...

**Pull request merged**